src/components/walls/BrickWall/BrickWall.tsx

```diff
@@ -0,0 +1,10 @@
+import React, {FC} from 'react';
+
+import style  from './BrickWall.module.scss';
+import brick from '../../../assets/images/walls/wall_1.svg'
```

Comment:
try to understand if the loading images will be delayed in this way or not. Try testing with some throttling and cleared caches.